newIDE/app/src/EventsSheet/ParameterFields/VariableField.js

```diff
@@ -55,6 +54,10 @@ type Props = {
   ...ParameterFieldProps,
   isObjectVariable: boolean,
   variablesContainers: Array<gdVariablesContainer>,
+  getVariableSourceFromVariableName: (
+    variableRootName: string,
+    projectScopedContainers: gdProjectScopedContainers
+  ) => VariablesContainer_SourceType,
```

Comment:
Let's call this:

```suggestion
  getVariableSourceFromIdentifierName: (
    identifierName: string,
    projectScopedContainers: gdProjectScopedContainers
  ) => VariablesContainer_SourceType,
```

to emphasis that we don't know if it's a variable name, a property name or a parameter name, and it could be a mix.